src/lib/y2network/hwinfo.rb

```diff
@@ -278,7 +278,11 @@ def used_mac
     # @param other [Hwinfo] Object to compare with
     # @return [Boolean]
     def ==(other)
```

Comment:
Why isn't equatable used here?

Reply:
I guess we can use: `eql_attr :hwinfo`

Reply:
I discarded this case because it uses hwinfo.compact for the comparison, so, we could use `eql_attr :hwinfo` but with
```
def eql_hash
  super.compact
end
```

Reply:
Sure, I would go with that approach.

Reply:
Adapted